tests/director/api/validation_test.go

```diff
@@ -53,7 +53,7 @@ func TestUpdateRuntime_Validation(t *testing.T) {
 
 	// THEN
 	require.Error(t, err)
-	assert.Contains(t, err.Error(), "name=cannot start with digit")
+	assert.EqualError(t, err, "graphql: Invalid data [name=Invalid data [reason=cannot start with digit]]")
```

Comment:
Can we get rid of the nested errors? The validation error message looks really weird to me right now.

Reply:
After the latest fixes, the error looks like

```graphql: Invalid data APIDefinitionInput [targetURL=Invalid data [reason=must be a valid URL]]```

Do you still think it should be changed?

Reply:
Well, with 3 invalid fields we will have 3x `Invalid data [reason=...` nested, I don't think it's pretty but if others don't have a problem with this I will understand.

Reply:
In this case, there is a bug in validtor, because the validator return `apperror`

Reply:
Now it looks like that:

`"graphql: Invalid data APIDefinitionInput [targetURL=must be a valid URL]"`